documentation/course/e2/README.md

```diff
@@ -2047,7 +2047,36 @@ TODO @jakubzehner: Dodać diagram pakietów, opis architektury i endpointy.
 
 ## Płatność
 
-TODO @piterek130: Dodać diagram pakietów, opis architektury i endpointy.
+TODO @piterek130: Dodać diagram pakietów, opis architektury.
+
+### API
+
+#### API publiczne
+
+| **Rola**                 | **Metoda** | **Endpoint**                   | **Wymagania**                | **Opis**                                    |
+| ------------------------ | ---------- | ------------------------------ | ---------------------------- | ------------------------------------------- |
+| `passenger`              | `GET`      | `/ext/v1/cards`                | `PAY/19`                     | Pobranie listy kart płatniczych.            |
+| `passenger`              | `POST`     | `/ext/v1/cards`                | `PAY/02`                     | Dodanie nowej karty płatniczej.             |
+| `passenger`              | `PUT`      | `/ext/v1/cards/:id`            | `PAY/20`                     | Zaktualizowanie danych karty płatniczej.    |
+| `passenger`              | `DELETE`   | `/ext/v1/cards/:id`            | `PAY/03`                     | Usunięcie karty płatniczej.                 |
+| `passenger`              | `POST`     | `/ext/v1/wallet/add-funds`     | `PAY/07`                     | Doładowanie portfela.                       |
+| `passenger`              | `GET`      | `/ext/v1/wallet`               | `PAY/08`                     | Pobranie stanu portfela.                    |
+| `passenger`              | `GET`      | `/ext/v1/wallet/history`¹      | `PAY/09`                     | Pobranie historii doładowań portfela.       |
+| `passenger`              | `GET`      | `/ext/v1/transations`¹         | `PAY/10`                     | Pobranie historii transakcji.               |
+| `passenger`, `inspector` | `GET`      | `/ext/v1/fines`¹               | `PAY/17`                     | Pobranie listy mandatów.                    |
+| `passenger`              | `GET`      | `/ext/v1/fines/:id`            | `PAY/18`                     | Pobranie informacji o mandacie.             |
+| `passenger`              | `POST`     | `/ext/v1/payment`              | `PAY/14`, `PAY/15`, `PAY/16` | Realizacja płatności.                       |
+| `inspector`              | `POST`     | `/ext/v1/fines`                | `PAY/19`                     | Wystawienie mandatu.                        |
+| `inspector`              | `PUT`      | `/ext/v1/fines/:id`            | `PAY/20`                     | Zmiana statusu mandatu.                     |
+
```

Comment:
**issue:** I still don't see a way to pay for a fine? How about `POST /ext/v1/fines/:id/pay`?